src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -76,6 +77,28 @@ const extraRequiredCollateralTokenAmount = newMonetaryAmount(
   true
 );
 
+const getTokenFieldHelperText = (message?: string) => {
+  switch (message) {
+    case 'no_issuable_token_available':
+      return (
+        <Trans i18nKey='no_issuable_token_available'>
+          Oh, snap! All iBTC minting capacity has been snatched up. Please come back a bit later, or{' '}
+          <InterlayLink
+            className='underline'
+            target='_blank'
+            rel='noreferrer'
+            href='https://docs.interlay.io/#/vault/overview'
```

Comment:
Is this the correct link?

Reply:
Should we add this to our constants file?

Reply:
I'm not sure whether it's correct but I think we should add this to "constants" file. Specifically `/src/config/links.ts`